index.html

```diff
@@ -0,0 +1,309 @@
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-4">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-5">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-6">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg btn-dark-yellow" href="#">See Project</a>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-section padding-left-right">
+        <div class="about-box">
+            <div class="about-text-box">
+                <h2 class="about-heading">About me</h2>
+                <p class="about-description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] In the mobile view, I noticed that your `About section` looks different than how it was before and how it is in the Figma design. SO kidly fix that as well :wink:

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/009723d7-0ff6-4118-a1f9-6028bd9ee12c) | ![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/c29cf05a-8f5b-4edc-bb7f-4e6902b1e6a4) |